solutions/silver/usaco-835.mdx

```diff
@@ -4,6 +4,25 @@ source: USACO Silver 2018 Open
 title: Lemonade Line
 author: Mrinall Umasudhan, Ryan Chou
 ---
+<Spoiler title="Hint 1">
+
+Consider how you can process the cows to minimize the number of cows joining the line. Think about that possible situations where a cow would be willing to wait behind the others.
+
+</Spoiler>
+
+<Spoiler title="Hint 2">
+
+This problem is best solved with a greedy approach. Consider which data structures will be useful in being able to get the cow that is the most willing to wait in line.
```

Comment:
people presumably come here from the greedy module so...

Reply:
Yeah just remove the first part of the sentence